test/unit/cache/update.test.js

```diff
@@ -0,0 +1,42 @@
+jest.mock('../../../app/cache/get')
+const { get: mockGet } = require('../../../app/cache/get')
+
+jest.mock('../../../app/cache/set')
+const { set: mockSet } = require('../../../app/cache/set')
+
+const { NAME } = require('../../mocks/cache/name')
+const { KEY } = require('../../mocks/cache/key')
+const { VALUE } = require('../../mocks/cache/value')
+
+const { update } = require('../../../app/cache/update')
+
+describe('cache set', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+    mockGet.mockResolvedValue({})
+  })
+
+  test('should get existing item', async () => {
+    await update(NAME, KEY, VALUE)
+    expect(mockGet).toHaveBeenCalledWith(NAME, KEY)
+  })
+
+  test('should set value as is if no existing item', async () => {
```

Comment:
Could we change this to be something like 'should set value as is if no item exists'?